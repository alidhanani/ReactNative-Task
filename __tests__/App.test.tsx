import React from 'react';
import renderer from 'react-test-renderer';
import { Text } from 'react-native';

describe('Text component', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Text>Hello World!</Text>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
