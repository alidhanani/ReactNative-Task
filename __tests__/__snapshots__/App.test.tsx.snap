// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text component renders correctly 1`] = `
<Text>
  Hello World!
</Text>
`;
